pyproject.toml

```
@@ -20,18 +20,18 @@ license = "MIT-0"
 
 [tool.poetry.dependencies]
 python = "^3.7.4"
-aws-xray-sdk = "^2.8.0"
-fastjsonschema = "^2.14.5"
-boto3 = "^1.18"
-pydantic = {version = "^1.8.2", optional = true }
-email-validator = {version = "*", optional = true }
+aws-xray-sdk = { version = "^2.8.0", optional = true }
+fastjsonschema = { version = "^2.14.5", optional = true }
+pydantic = { version = "^1.8.2", optional = true }
+email-validator = { version = "*", optional = true }
```

```suggestion
email-validator = { version = "^1.3.0", optional = true }
```

Sins from the past. This will make our install slightly faster, since `*` will instruct pip to download all available options in PyPi before handling the dependency resolution.  That is, it'll download everything listed here before choosing one: https://pypi.org/project/email-validator/#history